recipes/rmm/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+export INSTALL_PREFIX=${CONDA_PREFIX}
+
+cd ${SRC_DIR}/rmm/python
+
+$PYTHON setup.py build_ext --inplace
+$PYTHON setup.py install
```

Comment:
The whole script here is not needed, and also the `setup.py install` part needs to be improved to avoid accidentally bringing in external dependencies. I'd suggest to remove this script and just add below to relevant sections in `meta.yaml`:
```yaml
  script_env:
    - INSTALL_PREFIX=${PREFIX}
  script:
    - cd rmm/python
    - {{ PYTHON }} -m pip install . --no-deps -vv
```